joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/RegexTests.scala

```diff
@@ -21,7 +22,7 @@ class RegexTests extends RubyCode2CpgFixture(withPostProcessing = false) {
       val tmpSource = tmpInit.source.asInstanceOf[Call]
       tmpSource.code shouldBe s"/h(el)lo/.match($expectedSubject)"
       tmpSource.name shouldBe "match"
-      tmpSource.methodFullName shouldBe "__core.Regexp.match"
+      tmpSource.methodFullName shouldBe Defines.DynamicCallUnknownFullName
```

Comment:
Should we not include a check to the type hint somewhere to ensure it is also working properly?